jira-macro/jira-macro-common/pom.xml

```diff
@@ -33,16 +33,17 @@
   <description>Common abstractions and utilities of all JIRA macros.</description>
   <properties>
     <checkstyle.suppressions.location>${basedir}/src/checkstyle/checkstyle-suppressions.xml</checkstyle.suppressions.location>
-    <xwiki.jacoco.instructionRatio>0.51</xwiki.jacoco.instructionRatio>
+    <xwiki.jacoco.instructionRatio>0.45</xwiki.jacoco.instructionRatio>
   </properties>
   <dependencies>
     <dependency>
       <groupId>org.jdom</groupId>
       <artifactId>jdom2</artifactId>
     </dependency>
     <dependency>
-      <groupId>org.apache.httpcomponents</groupId>
-      <artifactId>httpclient</artifactId>
+      <groupId>org.apache.httpcomponents.client5</groupId>
+      <artifactId>httpclient5</artifactId>
+      <version>5.2</version>
```

Comment:
Not sure why you need this upgrade, but I would put the version directly as a property of the root pom to ease future upgrades.

Reply:
The main reason to upgrade it, is that the Httpclient will be used by the external API of the `JIRAAuthenticator` and this API will be used by an external extension so after this it might be more complicated to upgrade it.